docs/config.md

```diff
@@ -200,6 +200,7 @@ due to lack of stable semantic convention.
 | ID        | Instrumented library                                                                                                            | Supported versions | Instrumentation type   | Status                                                                                                                            |
 |-----------|---------------------------------------------------------------------------------------------------------------------------------|--------------------|------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
 | `ILOGGER` | [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging) **Not supported on .NET Framework** | ≥8.0.0             | bytecode or source [1] | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
+| `LOG4NET` | [log4net](https://www.nuget.org/packages/log4net)                                                                               | ≥2.0.0 && < 4.0.0  | bytecode               | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
```

Comment:
Side note: The section should be "Logs bridges" or "Logs appenders" and not "Logs instrumentations". The "Logs instrumentations" may come in future. E.g. OTel may add Logs instrumentations for HTTP Client.